profile.html

```diff
@@ -1,25 +1,91 @@
 <!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="styles.css">
+<html lang="en" data-js="htmlGoesDark">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
     <title>Quizzen-mit-Wizzen</title>
-   
-  
-</head>
+    <script src="./index.js" defer></script>
+  </head>
 
-<body>
+  <body class="profile-body" data-js="bodyGoesDark">
+    <h1 class="header">Quizzen mit Wizzen</h1>
 
-    <h1>Quizzen mit Wizzen</h1>
+    <form action="Name" class="label-name">
```

Comment:
I think you nede to use the `name` attribute instead of `action`
- also name shall be small letters
` <form name="name" class="label-name">`